### PR TITLE
feature(cloud_monitor): add support for showing multiple projects

### PR DESCRIPTION
**Pull request by fruch**
now that we have multiple project for GCE, we need to show and report
them all

New project column for GCE instances:
image

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/utils/cloud_monitor/resources/__init__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 class CloudInstance:  # pylint: disable=too-few-public-methods,too-many-instance-attributes
     pricing = None  # need to be set in the child class
 
-    def __init__(self, cloud, name, instance_id, region_az, state, lifecycle, instance_type, owner, create_time, keep):  # pylint: disable=too-many-arguments
+    def __init__(self, cloud, name, instance_id, region_az, state, lifecycle, instance_type, owner, create_time, keep, project='N/A'):  # pylint: disable=too-many-arguments
```
Comment (fgelcer) [inline sdcm/utils/cloud_monitor/resources/__init__.py:10]: do we have now a default project?

Comment (fgelcer) [reply sdcm/utils/cloud_monitor/resources/__init__.py:10]: if we do, we should use it in here, instead of `N/A`

Comment (fruch) [reply sdcm/utils/cloud_monitor/resources/__init__.py:10]: yes, for gce (adding it in a diffrent PR), but this default is also for backends we don't have projects in, i.e. AWS

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by roydahan — APPROVED** [review]

**Pull request merged**